ui/src/app/shared/genericComponents/shared/formatter.ts

```diff
@@ -0,0 +1,18 @@
+import { formatNumber } from "@angular/common";
+
+export namespace Formatter {
+  export const FORMAT_WATT = (value: number) => {
+    // TODO apply correct locale
+    return formatNumber(value, 'de', '1.0-0') + " W";
```

Comment:
Just a thought. maybe for the future, or can be ignored.
Specifying locales shouldn't be a static string. types have to be used for safety.

Reply:
in the future yes